server.js

```diff
@@ -3,17 +3,21 @@
 const express = require('express');
 const cors = require('cors');
 const superagent = require('superagent');
+const pg = require('pg');
 
-require('dotenv').config();
+require('dotenv').config();//allows me to read the .env file
 
 /* ----------------- Setup Application Server ----------------- */
 
 const app = express();
 app.use(cors());
 
+const DATABASE_URL = process.env.DATABASE_URL;//add DATABASE_URL to .env
+const client = new pg.Client(DATABASE_URL);
+client.on('error', error => console.log(error));
+
```

Comment:
I encourage you to use comments more abundantly throughout to explain what your code is doing.